### ISSUE TITLE
Markdown does not render when using `--fullscreen`

### ISSUE THREAD

**Issue by dan-snelson** [bartreardon/Dialog#78]
First, thanks for all your hard work on Dialog; simply excellent.

In a single test with Dialog `1.9.2` running on macOS `12.3 (21E5212f)`,  Markdown renders as expected, except when using `--fullscreen`; see attached.

---

## Markdown Renders

```
/usr/local/bin/dialog --title 'Title Goes Here' --message "# Heading 1  \n## Heading 2  \n### Heading 3  \nText can include **bold** and _italics_\n  * point 1\n * point 2\n * point 3  \n\n[Getting Started with Dialog](https://github.com/bartreardon/Dialog/wiki)\n\n  \![Inline images are also supported](https://www.stockvault.net/data/2010/01/20/112821/preview16.jpg)"  --icon /System/Library/CoreServices/Finder.app --iconsize 128 --messagefont "size=14" --button1text "button1text" --button2text "button2text" --infobuttontext "View KB8675309 …" --infobuttonaction "https://servicenow.company.com/support?id=kb_article_view&sysparm_article=KB8675309" --timer 60 
```

<img width="932" alt="Screen Shot 2022-02-18 at 6 11 37 AM" src="https://user-images.githubusercontent.com/24623109/154690229-dd8dc0c6-d848-43a3-8e32-09d5aed84bf5.png">

---

## Markdown does _not_ render when using `--fullscreen`

```
/usr/local/bin/dialog --title 'Title Goes Here' --message "# Heading 1  \n## Heading 2  \n### Heading 3  \nText can include **bold** and _italics_\n  * point 1\n * point 2\n * point 3  \n\n[Getting Started with Dialog](https://github.com/bartreardon/Dialog/wiki)\n\n  \![Inline images are also supported](https://www.stockvault.net/data/2010/01/20/112821/preview16.jpg)"  --icon /System/Library/CoreServices/Finder.app --iconsize 128 --messagefont "size=14" --button1text "button1text" --button2text "button2text" --infobuttontext "View KB8675309 …" --infobuttonaction "https://servicenow.company.com/support?id=kb_article_view&sysparm_article=KB8675309" --timer 60  --fullscreen
```

<img width="1440" alt="Screen Shot 2022-02-18 at 6 12 44 AM" src="https://user-images.githubusercontent.com/24623109/154690432-63c39c23-c833-4d34-9ca0-c619cfa062c5.png">

**Comment by dan-snelson** [comment]
(Perhaps, just update the Wiki entry for [Full Screen](https://github.com/bartreardon/Dialog/wiki/Full-Screen) to advise users Markdown doesn't render when using `--fullscreen`.)

**Comment by bartreardon** [comment]
If it makes sense to have markdown in fullscreen mode then its easy enough to add it in. Im thinking of some other layout changes here in any case but its more of a basic view than the main dialog window. Its mostly modelled to give a reasonable approximation to jamfhelpers fullscreen mode.

**Comment by bartreardon** [comment]
@dan-snelson how would you feel about something like this?

Looking through my code, I did make a start on markdown for the fullscreen view but didn't finish it. (fullscreen is supposed to be a much simplified display after all)
I did place the text in a scroll view and also added a display for any timer present. I'm not sold on making the text scrollable as it's not obvious that the text here can be scrolled. To compensate for the lack of space in the fullscreen view I made the icon display at a maximum of 1/4 the screen height. 

https://user-images.githubusercontent.com/3598965/155521135-534eda81-e369-4832-94e6-700f242d7bea.mov

**Comment by dan-snelson** [comment]
One word: Sweetness!

LGTM, @bartreardon

**Comment by bartreardon** [comment]
fixed in [1.9.3](https://github.com/bartreardon/Dialog/releases/tag/v1.9.3)
